src/settings/BatchGroupConfigurationSettings/BatchGroupConfigurationSettings.js

```diff
@@ -6,14 +6,67 @@ import React, {
 import PropTypes from 'prop-types';
 
 import { LoadingPane } from '@folio/stripes/components';
+import {
+  useShowCallout,
+} from '@folio/stripes-acq-components';
 
-import { batchGroupsResource } from '../../common/resources';
+import { EXPORT_CONFIGURATIONS_API } from '../../common/constants';
+import {
+  batchGroupsResource,
+  credentialsResource,
+  exportConfigsResource,
+} from '../../common/resources';
+import { SCHEDULE_EXPORT } from './constants';
+import { saveExportConfig } from './utils';
 import BatchGroupConfigurationForm from './BatchGroupCongiurationForm';
 
 const BatchGroupConfigurationSettings = ({ mutator }) => {
   const [isLoading, setIsLoading] = useState(true);
   const [batchGroups, setBatchGroups] = useState();
   const [selectedBatchGroupId, setSelectedBatchGroupId] = useState();
+  const [exportConfig, setExportConfig] = useState();
+  const [credentials, setCredentials] = useState();
+  const showCallout = useShowCallout();
+
+  const fetchExportConfig = useCallback(
+    (id) => {
+      if (id) {
+        setIsLoading(true);
+        setExportConfig();
+        setCredentials();
+
+        const credentialsPromise = (exportConfigResp) => {
+          if (exportConfigResp.length) {
+            const exportConfigId = exportConfigResp[0].id;
+
+            setExportConfig(exportConfigResp[0]);
+            mutator.exportConfigId.update({ id: exportConfigId });
+
+            return mutator.credentials.GET({
+              path: `${EXPORT_CONFIGURATIONS_API}/${exportConfigId}/credentials`,
```

Comment:
you don't need this actually, since you've described path in the resource's manifest